docs/book/advanced_guide/step-operators.md

```diff
@@ -0,0 +1,255 @@
+---
+description: Execute individual steps in specialized environments.
+---
+
+# Run steps in specialized environments with step operators
+
+The step operator defers the execution of individual steps in a pipeline to
+specialized runtime environments that are
+optimized for Machine Learning workloads. This is helpful when there is a
+requirement for specialized cloud backends ✨
```

Comment:
The only emoji I found (or remember) inline in text, can probably remove that as well?